cmd/flags.go

```diff
@@ -102,8 +103,11 @@ func CreateFlags(defaultPath string) []cli.Flag {
 			Usage: "By setting this flag to true, disables the need to wait the propagation of the TXT record to all authoritative name servers.",
 		},
 		&cli.StringSliceFlag{
-			Name:  "dns.resolvers",
-			Usage: "Set the resolvers to use for performing recursive DNS queries. Supported: host:port. The default is to use the system resolvers, or Google's DNS resolvers if the system's cannot be determined.",
+			Name: "dns.resolvers",
+			Usage: "Set the resolvers to use for performing (recursive) CNAME resolving and apex domain determination." +
+				" For DNS challenge verification, the authoritative DNS server is queried directly." +
+				" Supported: host:port." +
+				" The default is to use the system resolvers, or Google's DNS resolvers if the system's cannot be determined.",
```

Comment:
Please also include an example. It's not clear how to provide multiple values for this setting. Do you provide a CSV or something else?

Reply:
If you run `lego --help` (or see [here](https://github.com/go-acme/lego/pull/1786/files#diff-fe0fb6c80906c79ff932549d18a30a1cff5846e1ac80a7d64a30baa965a0d74fR28), you'll find that you can just specify the `--dns.resolvers` flag multiple times:

```
$ lego --help

...

   --dns.resolvers value [ --dns.resolvers value ]
              Set the resolvers to use for performing (recursive) CNAME resolving and
              apex domain determination. For DNS challenge verification, the authoritative
              DNS server is queried directly. Supported: host:port. The default is to use
              the system resolvers, or Google's DNS resolvers if the system's cannot be
              determined.
```

(I've reformatted it a bit to cope with the restricted space imposed by Github.)

Reply:
Is that new? I have v4.8.0 (via https://github.com/void-linux/void-packages/blob/master/srcpkgs/lego/template) and I see:

```
> lego --help
**snip**
--dns.resolvers value        Set the resolvers to use for performing recursive DNS queries. Supported: host:port. The default is to use the system resolvers, or Google's DNS resolvers if the system's cannot be determined.
**snip**
```

Still, I think it would be good to state that the option can be repeated multiple times for multiple resolvers. This is the first time I can recall seeing this bracket syntax and it isn't super clear.

Reply:
Hm, curious. I'll have to investigate...

Reply:
Looks like we've updated the CLI argument parser library in Lego 4.9.0 from [2.3.0 to 2.14.0](https://github.com/urfave/cli/compare/v2.3.0...v2.14.0), which could explain differences in the output.

Reply:
👍